cumulus_library_data_metrics/__init__.py

```diff
@@ -1,3 +1,3 @@
 """Data Metrics study for Cumulus Library"""
 
-__version__ = "6.1.0"
+__version__ = "7.0.0"
```

Comment:
Major bump because this PR splits the previous c_us_core_v4_count DocumentReference table into two (for CUBE performance reasons), which could affect any consumers.